migrations/1706173396405-account-and-user-permission-update.js

```diff
@@ -0,0 +1,123 @@
+const mongoose = require('mongoose');
```

Comment:
we don't have mongoose migrations anymore, we use instead mikro-orm migrations
look at code in apps/server/src/migrations/mikro-orm
https://documentation.dbildungscloud.dev/docs/schulcloud-server/Migrations